src/styl/site.styl

```diff
@@ -130,84 +91,6 @@ $color-blue = rgb(84, 144, 235)
 
 // ******************************************************
 
-// BACKGROUNDS
-
-.background-black
-  background-color: var(--color-black-soft)
-
-.background-white
-  background-color: var(--color-white-soft)
-
-.background-gray
-  background-color: var(--color-gray)
-
-.background-light-gray
-  background-color: var(--color-gray-light)
-
-.background-red
-  background-color: var(--color-red)
-
-.background-yellow
-  background-color: var(--color-yellow)
-
-.background-green
-  background-color: var(--color-green)
-
-.background-green-neon
-  background-color: var(--color-green-neon)
-
-.background-light-blue
-  background-color: var(--color-blue-light)
-
-.background-neon-blue
-  background-color: var(--color-blue-neon)
-
-.background-teal
-  background-color: var(--color-teal)
-
-.background-lilac
-  background-color: var(--color-lilac)
-
-.background-light-purple
-  background-color: var(--color-lilac)
-
-.background-acfm-pink
-  background-color: #FC16CB
-
-.background-cover-image
-  position: relative
-  background-size: cover
-  background-position: center
-  background-repeat: no-repeat
-
-.background-cover-overlay
-  position: absolute
-  top: 0
-  width: 100%
-  height: 100%
-
-.background-support-texture, .background-support-texture-alt
-  background-size: cover
-  background-position: center
-  background-repeat: no-repeat
-  background-blend-mode: overlay, normal
-
-.avif, .webp
```

Comment:
because these rules relate to a specific file path of the support texture they need to stay on the main repo. (Unless we decide that we should use the library to also have shared assets, which might not be a bad idea in general, but would require a bit more work for this PR)